agents-for-bedrock/use-case-examples/hr-assistant/agent/knowledge-base-assets/links.txt

```diff
@@ -0,0 +1,2 @@
+401K contribution limit: The Internal Revenue Service announced today that the amount individuals can contribute to their 401(k) plans in 2024 has increased to $23,000.
```

Comment:
same here: where is this data coming from?

Reply:
Data is changing to leaglly approved data